grove/alpha/simon/utils.py

```diff
@@ -0,0 +1,70 @@
+import numpy as np
+
+PADDED_BINARY_BIT_STRING = "{0:0{1:0d}b}"
+
```

Comment:
I think this file should be pulled out of `simon`. I know you likely just put it here because this is where you're developing, but we're going to need to stitch our two util files together, and they are for all the `alpha` modules atm.

Reply:
I agree. The intention is to unify all the utils in a separate PR. I don't want to enter merging hell and coordination hell to get all the PRs in. This will make it easier to consolidate it later on